apps/labs/posts/re-engineering-cicd-pipelines-for-scipy.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'Re-Engineering CI/CD pipelines for SciPy'
+published: October 11, 2021
+author: harsh-mishra
+description: 'In this blog post I talk about the projects and my work during my internship at Quansight Labs. My efforts were geared towards re-engineering CI/CD pipelines for SciPy to make them more efficient to use with GitHub Actions.'
+category: [Developer workflows]
+featuredImage:
+  src: /posts/re-engineering-cicd-pipelines-for-scipy/feature.png
+  alt: 'The picture displays a logo of Quansight Labs on the left and a logo of SciPy on the right. It signifies the primary purpose of the project at Quansight Labs to re-engineer the GitHub Actions CI for the SciPy and lay down the further scope for developing an entire CI matrix for build, test and release.'
+hero:
+  imageSrc: /posts/re-engineering-cicd-pipelines-for-scipy/hero.png
+  imageAlt: 'The picture displays a logo of Quansight Labs on the left and a logo of SciPy on the right. It signifies the primary purpose of the project at Quansight Labs to re-engineer the GitHub Actions CI for the SciPy and lay down the further scope for developing an entire CI matrix for build, test and release.'
+---
+
+In this blog post I talk about the projects and my work during my internship at Quansight Labs. My efforts were geared towards re-engineering CI/CD pipelines for SciPy to make them more efficient to use with GitHub Actions. I also talk about the milestones that I achieved, along with the associated learnings and improvements that I made.
+
+This blog post would assume a basic understanding of CI/CD and GitHub Actions. I will also assume a basic understanding of Python and the SciPy ecosystem.
+
+<p align="center">
```

Comment:
I think we can remove this image now that it's the hero image.

Reply:
OK! Removed. I think I missed it because it was a few paragraphs down instead of right on top like some other posts.

Reply:
OK! Image removed and PR merged.

Reply:
Github says this PR needs approval. Could you approve it now? Or should I merge it myself, since this a minor change?